components/camel-kafka/src/main/java/org/apache/camel/component/kafka/KafkaFetchRecords.java

```diff
@@ -297,6 +297,7 @@ private void subscribe() {
     }
 
     protected void startPolling() {
+        // not sure what we are using this for
         long partitionLastOffset = -1;
```

Comment:
saw this and it is used in Polling Strategy but I didn't see where we changed it